### PR TITLE
Openshift: Deploy kube-controller-manager

### PR DESCRIPTION
**Pull request by alvaroaleman**
**What this PR does / why we need it**:
Deploys the kube-controller-manager for openshift 4

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

/assign @mrIncompetent

### CODE REVIEW

**Comment by alvaroaleman** [comment]
@mrIncompetent PTAL

File: api/pkg/controller/openshift/resources/kube_controller_manager.go [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+				}
+				vars := struct {
+					CACertPath            string
+					CAKeyPath             string
+					ClusterCIDR           string
+					ServiceAccountKeyFile string
+					ServiceCIDR           string
+				}{
+					CACertPath:            "/etc/kubernetes/pki/ca/ca.crt",
+					CAKeyPath:             "/etc/kubernetes/pki/ca/ca.key",
```
Comment (mrIncompetent) [inline api/pkg/controller/openshift/resources/kube_controller_manager.go]: If this is static, why do we template that?

**Comment by alvaroaleman** [comment]
@mrIncompetent removed the static paths to a const, PTAL

**Pull request merged**